src/components/AddMoney/views/RhinoDeposit.view.tsx

```
@@ -0,0 +1,166 @@
+                            <p className="text-sm text-grey-1">Min deposit for EVM networks</p>
+                        </div>
+
+                        <p className="text-sm font-medium text-grey-1">{depositAddressData.minDepositLimitUsd} USD</p>
+                    </div>
+
+                    {chainType === 'EVM' && (
+                        <Card className="space-y-2 p-4">
+                            <h3 className="text-sm font-bold text-black">Supported EVM networks</h3>
+                            <div className="flex flex-wrap gap-2">
+                                {SUPPORTED_EVM_CHAINS.map((chain) => (
+                                    <ChainChip chainName={chain} tokenSymbol={CHAIN_LOGOS[chain]} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)